### PR TITLE
add katib releaser

### PR DESCRIPTION
**Pull request by YujiOshima**
I want to release 7 images for Katib.

* vizier-core
* suggestion-random
* suggestion-grid
* suggestion-hyperband
* suggestion-bayesianoptimization
* earlystopping-medianstopping
* katib-frontend

@kunmingg  Please take a look.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1102)

### CODE REVIEW

**Comment by kunmingg** [comment]
@YujiOshima 
Did you forget to include the actual release ks app? (should be at kubeflow/kubeflow/releasing/katib, like pytorch)

**Comment by YujiOshima** [comment]
@kunmingg Do you mean [pytorch-operator.jsonnet](https://github.com/kubeflow/kubeflow/blob/master/releasing/pytorch-operator/components/pytorch-operator.jsonnet) ?
I added katib.jsonnet.

File: releasing/katib/components/params.libsonnet [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  components: {
+    // Component-level parameters, defined initially from 'ks prototype use ...'
+    // Each object below should correspond to a component in the components/ directory
+    katib: {
+      bucket: "kubeflow-releasing-artifacts",
+      cluster: "kubeflow-releasing",
+      dockerfile: "Dockerfile",
+      dockerfileDir: "kubeflow/pytorch-operator",
+      extra_args: "/mnt",
+      extra_repos: "kubeflow/testing@HEAD;kubeflow/pytorch-operator@HEAD",
```
Comment (kunmingg) [inline releasing/katib/components/params.libsonnet]: Should be katib repo

**Comment by kunmingg** [comment]
Presubmit failed on format check, can use jsonnet fmt following [docs](https://github.com/kubeflow/kubeflow/blob/master/docs_dev/releasing.md#create-release-workflow)
Otherwise lgtm.

**Comment by YujiOshima** [comment]
@kunmingg Thank you for your advice!
I updated. PTAL

**Pull request merged**